go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/vshn/appcat-charts
 
-go 1.25
+go 1.24
```

Comment:
why downgrade?

Reply:
why downgrade production code to appease to dev? I would upgrade devcontainer at this point not downgrade here

Reply:
Reasons I opted for this:

- Having our go versions all over the place doesn't help either
- The only reason for go in this repo is `helm-docs` which is hardly in the critical path of any production logic
- We should always use the devcontainer to ensure consistency

Having said that I agree that we should update the container. I'm testing go 1.25 as we speak, if everything is okay I'll merge and revert the version here.